cli/utilities/utils.py

```diff
@@ -320,3 +322,55 @@ def set_service_state(node, service_id, state):
     if err:
         return False
     return True
+
+
+def bring_node_offline(node, secondary_node, interface="eth0", timeout=100):
+    """
+    Brings a node offlibe by making network interface down for a defined time
+        Args:
+            node (ceph): Node at which the interface has to be bought down
+            secondary_node (ceph) : Another node in the cluster to ping and verify
```

Comment:
Why secondary_node ? Can' we execute ping command from executor node (node on which CephCI code is installed) ?

Reply:
Secondary node is required to perform a ping to the down node. Because the `node` will be down as the interface is set to down and we will not be able to connect to it. That's the reason why a second node is taken.